src/flamegpu/gpu/CUDAEnsemble.cu

```diff
@@ -316,6 +317,13 @@ int CUDAEnsemble::checkArgs(int argc, const char** argv) {
             }
             continue;
         }
+        // -s/--standby Disable the blocking of standby
+        if (arg.compare("--standby") == 0 || arg.compare("-s") == 0) {
```

Comment:
I'm not keen on `-s`, as `-s` means steps on the simulation cli. We don't need short options for everything. 

Also seem to have missed adding it to `CUDAEnsemble::printHelp`.

Reply:
Fair, will address both those.